test/IntegrationTests/BuildTests.DistributionStructure_alpine-linux.verified.txt

```diff
@@ -71,6 +71,9 @@
   /store/x64/net7.0/microsoft.extensions.options.configurationextensions/8.0.0/lib/net8.0/Microsoft.Extensions.Options.ConfigurationExtensions.dll,
   /store/x64/net7.0/microsoft.extensions.options/8.0.0/lib/net8.0/Microsoft.Extensions.Options.dll,
   /store/x64/net7.0/microsoft.extensions.primitives/8.0.0/lib/net8.0/Microsoft.Extensions.Primitives.dll,
+  /store/x64/net7.0/opentelemetry.api.providerbuilderextensions/1.7.0/lib/net6.0/OpenTelemetry.Api.ProviderBuilderExtensions.dll,
```

Comment:
Do you understand why this assembly shows up under `net6.0` here, but, under `net8.0` on the line above for `net6.0` apps?

Reply:
Missing roll forward - see d223f5656e9d79a367ad77f3dc782244a42d867a